src/Components/Hero/Hero.scss

```diff
@@ -0,0 +1,65 @@
+﻿$curiosityThemeColor: grey;
+$spiritThemeColor: grey;
+$opportunityThemeColor: grey;
```

Comment:
These seem like nice things to have in the shared constants

Reply:
although perhaps this hasn't been merged in yet?

Reply:
Yeah I'm waiting for the constants file to get merged into master and then can be swapped in